### DISCUSSION TITLE
请问uosc的文件浏览器如何默认显示所有驱动器

### DISCUSSION THREAD

**Discussion by redomCL** [dyphire/mpv-config#110] [Q&A] [answered]
请问这行要怎么写default_directory=
才能默认以这个视图弹出，非常感谢
![QQ截图20240813121705](https://github.com/user-attachments/assets/07e3c056-9eb0-49e7-891c-9c83ad8d8d7d)

**Comment by dyphire** [comment]
做不到，因为 uosc 不支持，可以修改实现，但不确定 linux 上相关应该如何处理。建议在 uosc 上游打开相应的功能请求

**Reply by redomCL** [reply]
感谢解答

**Reply by redomCL** [reply]
正好借着这个贴说一句，建议uosc把步进时的进度条也加上算了，音量都有了，我尝试向uosc提交一下功能请求
![QQ图片20240819172558](https://github.com/user-attachments/assets/81caac75-9c82-4690-a378-1a34ff6d0e8f)

**Reply by redomCL** [reply]
看了一下，现在好像也没办法通过conf简单设置，让下面的进度条在步进时暂时出现吧
![2](https://github.com/user-attachments/assets/842868d5-5f2e-4b3f-b477-ca6c2df2f470)

**Reply by dyphire** [reply]
我觉得挺丑的，没必要。而且 mpv 默认的 osd-bar 都没有为帧步进步退添加进度条提示说明官方也认为没这个必要

**Reply by redomCL** [reply]
你这么一说，感觉uosc作者起初就没有这个意愿，我还是只提交一下Windows下显示所有驱动器的功能请求吧

**Comment by dyphire** [comment, marked as answer]
uosc 上游已实现此功能，关闭
